start_esp/start_esp.py

```diff
@@ -939,6 +940,11 @@ def make_argparser():
     parser.add_argument('--jwks_cache_duration_in_s', default=None, type=int, help='''
         Specify JWT public key cache duration in seconds. Default is 5 minutes.''')
 
+    parser.add_argument('--redirect_authorization_url', action='store_true',
```

Comment:
How about `jwt_redirect_authorization_url`? 

The current name just reminds me of service control check authz.

Reply:
Done